### ISSUE TITLE
Doesn't support raw ident

### ISSUE THREAD

**Issue by nhynes** [Goncalerta/proc-quote#10]
The following works in `quote` but fails in `proc_quote`:

```rust
#[test]
fn test_raw_ident() {
    quote!( r#the );
}
```

with the error
```
thread 'main' panicked at '"r#the" is not a valid Ident'
stack backtrace:
   7: proc_macro2::fallback::validate_term
   8: proc_macro2::fallback::Ident::_new
   9: proc_macro2::fallback::Ident::new
  10: proc_macro2::imp::Ident::new
  11: proc_macro2::Ident::new
  12: proc_quote::__rt::append_ident
```

**Comment by nhynes** [comment]
related: https://github.com/alexcrichton/proc-macro2/issues/140

**Comment by Goncalerta** [comment]
Thanks a lot for the report. 
Just fixed it in 60bd96e.

**Comment by nhynes** [comment]
Thanks a lot for the quick fix!
